### DISCUSSION TITLE
How to set up ComfyUI directory as backend?

### DISCUSSION THREAD

**Discussion by Exxb** [Stability-AI/StableSwarmUI#460] [Q&A] [answered]
I want to try out Swarm, but I already have all my models etc. in the ComfyUI directory. What do I need to do in Swarm to point my "Backends" to the directory in the ComfyUI folder that will let me access what I have installed without having to duplicate them into the Swarm folder?

**Comment by mcmonkey4eva** [comment]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2

**Reply by Exxb** [reply]
I'm trying to be able to use both the ComfyUI front end and the SwarmUI front end at once, to see which I like. Is it possible to do this? Do I need to relocate all my Comfy files to the Swarm directory instead?

**Reply by mcmonkey4eva** [reply]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2

**Reply by kalebbroo** [reply]
> I followed all the instructions, ran migrate-windows.bat, restarted, refreshed, etc., and I still didn't see any models loaded into SwarmUI.

What version does it now say you are at?

**Reply by Exxb** [reply]
v0.9.3.1

**Reply by mcmonkey4eva** [reply]
This is the Stability AI repo which has not been used in 6 months, use the SwarmUI repo not this.

**Reply by Exxb** [reply]
I followed the migration guide and clicked on the link that took me to https://github.com/mcmonkeyprojects/SwarmUI. I downloaded the install-Windows.bat file from there, and when I ran it, the version was still v0.9.3.1. Am I looking in the wrong place for the current version?

**Reply by mcmonkey4eva** [reply, marked as answer]
Look at the address bar at the top of your screen as you're posting here. You are posting in entirely the wrong place

**Reply by Exxb** [reply]
Sorry, I've been misunderstanding what you're trying to say. I'll post my question on the other page.
